src/signup/signup.service.spec.ts

```diff
@@ -80,7 +80,11 @@ describe("SignupService", () => {
 
     const sendEmailFunc = jest.spyOn(mailService, "sendTemplatedEmail");
 
-    const user = await service.signupUserEmailAndPassword(signupPayload, "/");
+    const user = await service.signupUserEmailAndPassword(
+      signupPayload,
+      "https://tamudatathon.com",
+      "/"
+    );
     expect(user).toBeDefined();
     expect(user.email).toBe(signupPayload.email);
     expect(sendEmailFunc).toHaveBeenCalled();
```

Comment:
Can we somehow test the URLs that are added to the email? (At least add tests to cover the case I described in the other comment).

Reply:
Probably not as the function is called with the rendered HTML, but I can make the `getVerificationLink` function public and add tests for it

Reply:
@phulsechinmay Nah don't worry about it